src/docs/release-notes/R19/R19.mdx

```diff
@@ -0,0 +1,35 @@
+### Figma Stickersheet
+
+-   New Design Pattern Stickersheet (TODO: Link) with examples of all of our [Design Patterns](/patterns).
```

Comment:
If you want to add more details — in DSM, we added:
* Spinners, progress bars, skeletons
* Those Eric icons
* Some panel components for data visualizations
* TODO: interactive components for buttons and form controls, might get taken out
* More pager components (created by Mariana while she worked on user auth workflow)
* Added steppers, horizontal and vertical
* Typography change, subtitle 1 font-weight 600
* All icons are now swappable
* Table components (table cell, table header, etc)

(Doesn't have to be this detailed, but I think at least some efforts are worth mentioning here)

When I publish design patterns and components, I will need to add changelog notes to them. I think it would be beneficial to mirror what's mentioned on this page to Figma's change log

Reply:
I would expect Figma's release notes to be more detailed than Doc-It. Doc-it is kind of a summary of the main highlights across the entire release, whereas Figma should go into the details of the changes there. Kind of like the difference between doc-it release notes and the changelog for individual packages.